substrate/utils/frame/benchmarking-cli/src/overhead/command.rs

```diff
@@ -208,7 +208,7 @@ fn create_inherent_data<Client: UsageProvider<Block> + HeaderBackend<Block>, Blo
 		let parachain_validation_data_provider = MockValidationDataInherentDataProvider::<()> {
 			para_id: ParaId::from(*para_id),
 			current_para_block_head: Some(header.encode().into()),
-			relay_offset: 1,
+			relay_offset: 0,
```

Comment:
review in progress: trying to figure out why this is needed

Reply:
Its a preparation for #9712 , we will disallow parachain slots which are "earlier" than relay chain slot. So the `relay_offset` of 1 will have relay chain slot 1 and para slot 0, which will error. Both 0 works though.

Reply:
Nice! Makes sense, thank you.